pages/common/comm.md

```diff
@@ -0,0 +1,24 @@
+# comm
+
+> Select or reject lines common to two files. Both files must be sorted and share collation.
```

Comment:
> sorted by the same criteria

I'm not sure. To make it simpler, I would rather drop the collation part altogether and hope that people will discover the difference between `LC_ALL=C sort` and `LC_ALL=en_US.UTF-8 sort` on their own.

So my suggestion would be:

> Select or reject lines common to two files. Both files must be sorted.